azerothcore/azerothcore-wotlk
# Scripts/Spells: Why double implementation Nurturing Instinct

## CHANGES PROPOSED:
-  Why double implementation of the talent Nurturing Instinct in SpellAuraEffect and spell_druid????

## TESTS PERFORMED:
- Build and test in game



## HOW TO TEST THE CHANGES:

- Create druid
- Learn talent Nurturing Instinct
- Use heal in form cat and see

## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
